src/main/webapp/app/entities/participation/participation.component.html

```diff
@@ -130,7 +139,28 @@ <h2>
                     </ng-template>
                 </ngx-datatable-column>
 
-                <ngx-datatable-column prop="student" [minWidth]="120">
+                <ngx-datatable-column [minWidth]="100" [width]="110">
+                    <ng-template ngx-datatable-header-template>
+                        <span class="datatable-header-cell-wrapper d-inline-block w-100 text-center" (click)="controls.onSort('submissionCount')">
+                            <span class="datatable-header-cell-label bold sortable" jhiTranslate="artemisApp.exercise.submissionCount">
+                                Submissions
+                            </span>
+                            <fa-icon [icon]="controls.iconForSortPropField('submissionCount')"></fa-icon>
+                        </span>
+                    </ng-template>
+                    <ng-template ngx-datatable-cell-template let-value="row">
+                        <div class="text-center">
+                            <div *ngIf="exercise.isAtLeastInstructor; else displayTextOnly">
```

Comment:
You probably can remove the div here and do `ngIf` right on the `<a>` tag to make the structure leaner

Reply:
Good point, done.